source/integrate-with-integration-environment/process-identity-information.html.md.erb

```diff
@@ -51,11 +51,22 @@ Content-Type: application/json
       "addressCountry": "GB",
       "validUntil": "2022-01-01"
     }
+  ]
+},
+  "https://vocab.account.gov.uk/v1/drivingPermit": [
```

Comment:
The fields for the drivingPermit claim have changed slightly now. The actual drivingPermit object we receive from the VC contains a few fields that we are now no longer including on our `drivingPermit` claim. Ticket here for the work that has just been done to remove some of the claims - https://govukverify.atlassian.net/browse/PYIC-2071

Reply:
Thanks, updated!